releases/release-1.19/README.md

```diff
@@ -0,0 +1,94 @@
+# Kubernetes 1.19
+
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.19/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.19/release_team.md)
+* [Meeting Minutes](http://bit.ly/k8s119-releasemtg) (join [kubernetes-sig-release@] to receive meeting invites)
```

Comment:
http://bit.ly/k8s119-releasemtg doesn't work, same for retro, enhancements doc, etc.

Reply:
@alejandrox1 I created the short link for the 1.19 Enhancements tracking sheet. The link should work now.

cc: @onlydole

Reply:
I will work on the permissions bit as the team gets stabilized.

Reply:
Thank you, @palnabarun!